content/en/guide/v10/typescript.md

```diff
@@ -56,13 +57,13 @@ If you use TypeScript within a Babel toolchain, set `jsx` to `preserve` and let
 
 Rename your `.jsx` files to `.tsx` for TypeScript to correctly parse your JSX.
 
-## Typing components
+## Typing Components
 
-There are different ways to type components in Preact. Class components have generic type variables to ensure type safety. TypeScript sees a function as functional component as long as it returns JSX. There are multiple solutions to define props for functional components.
+There are different ways to type components in Preact. Class components have generic type parameters to ensure type safety. TypeScript sees a function as functional component as long as it returns JSX. There are multiple solutions to define props for functional components.
 
-### Function components
```

Comment:
They are "Functional Components" and "Class Components" in other page.
https://github.com/preactjs/preact-www/blob/922098858eba2769900cc909b18b7b27e5487a49/content/en/guide/v10/components.md#functional-components